commit 630e972bc405dfcf5897ca9241bae3b6be1f5740
Author: Adrian Hunter <adrian.hunter@intel.com>
Date:   Thu Nov 22 16:04:56 2018 +0200

    perf test: Fix perf_event_attr test failure
    
    [ Upstream commit 741dad88dde296999da30332157ca47f0543747d ]
    
    Fix inconsistent use of tabs and spaces error:
    
      # perf test 16 -v
      16: Setup struct perf_event_attr                          :
      --- start ---
      test child forked, pid 20224
        File "/usr/libexec/perf-core/tests/attr.py", line 119
          log.warning("expected %s=%s, got %s" % (t, self[t], other[t]))
                                                                     ^
      TabError: inconsistent use of tabs and spaces in indentation
      test child finished with -1
      ---- end ----
      Setup struct perf_event_attr: FAILED!
    
    Signed-off-by: Adrian Hunter <adrian.hunter@intel.com>
    Cc: Jiri Olsa <jolsa@redhat.com>
    Link: http://lkml.kernel.org/r/20181122140456.16817-1-adrian.hunter@intel.com
    Signed-off-by: Arnaldo Carvalho de Melo <acme@redhat.com>
    Signed-off-by: Sasha Levin <sashal@kernel.org>

diff --git a/tools/perf/tests/attr.py b/tools/perf/tests/attr.py
index ff9b60b99f52..44090a9a19f3 100644
--- a/tools/perf/tests/attr.py
+++ b/tools/perf/tests/attr.py
@@ -116,7 +116,7 @@ class Event(dict):
             if not self.has_key(t) or not other.has_key(t):
                 continue
             if not data_equal(self[t], other[t]):
-		log.warning("expected %s=%s, got %s" % (t, self[t], other[t]))
+                log.warning("expected %s=%s, got %s" % (t, self[t], other[t]))
 
 # Test file description needs to have following sections:
 # [config]
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1150_630e972bc405dfcf5897ca9241bae3b6be1f5740.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1150_630e972bc405dfcf5897ca9241bae3b6be1f5740.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
